ansible/ncn-storage_nodes.yml

```diff
@@ -62,8 +55,3 @@
       vars:
         ssh_keys_username: 'root'
 
-    # Distribute SSH keys for the root user
-    - role: csm.ssh_keys
```

Comment:
Was this role also mistakenly being called twice in this playbook?

Reply:
Yes, it seems that it was